### ISSUE TITLE
About five attributes

### ISSUE THREAD

**Issue by huiyuan414** [wrk226/pytorch-multimodal_sarcasm_detection#5]
Hi, I am trying to run your code. But I have a problem in getting the attribute modality.
It seems that there is not the trained predictor in this repository.
Would you please explain how to train and which specific dataset and labels are used for training?
Thanks!

**Comment by wrk226** [comment]
You can find the image data from [here](https://github.com/headacheboy/data-of-multimodal-sarcasm-detection).
I didn't share the well-trained model because it doesn't take a long time to train it.
In order to train the model and understand the code, you may start from train.ipynb.

**Comment by huiyuan414** [comment]
> You can find the image data from [here](https://github.com/headacheboy/data-of-multimodal-sarcasm-detection). I didn't share the well-trained model because it doesn't take a long time to train it. In order to train the model and understand the code, you may start from train.ipynb.

Thanks. I have trained the model to detect sarcasm.
 But we have some problems to "predict" because when a new picture comes, how can we get five attributes like "img_to_five_words.txt"? 
 In other words, how can we train the ResNet in the procedure of attribute modality? The paper said it is fine-tuned through the COCO dataset with another ResNet model, but we did not find it:(

**Comment by wrk226** [comment]
> Thanks. I have trained the model to detect sarcasm. But we have some problems to "predict" because when a new picture comes, how can we get five attributes like "img_to_five_words.txt"? In other words, how can we train the ResNet in the procedure of attribute modality? The paper said it is fine-tuned through the COCO dataset with another ResNet model, but we did not find it:(

You are right, the author didn't share that model. When I tested the model on a new image I manually wrote five attributes for it... You may ask the author from the official repo or you may use any of the "Multi-class Classification" models for it.
